.gitignore

```diff
@@ -43,7 +43,7 @@ Makefile-pl
 config.cache
 config.status
 config.nice
-config.h
+/config.h
```

Comment:
Do we need this line? I don't have this file on my Mac and Ubuntu.

Reply:
you're right! We don't. Removed.